### DISCUSSION TITLE
Supporting @pixi/events

### DISCUSSION THREAD

**Discussion by zOadT** [michalochman/react-pixi-fiber#282] [General]
Hey! First of all, thank you very much for this package.

Pixi.js will [replace the interaction plugin](https://github.com/pixijs/pixijs/discussions/7833) with the events plugin in Pixi v7.

So this isn't a urgent issue, but I tried to test out react-pixi-fiber with the event plugin and run into a problem, see [this codesandbox](https://codesandbox.io/s/react-pixi-fiber-demo-rotating-bunny-forked-zxyu9v?file=/index.js). The problem is, that the `mousedown` etc properties got removed in the events plugin and therefore dont work anymore. Of couse you can still set listeners by using references, but thats pretty cumbersome.

As I said, this is not urgent, but are there any plans to add props like `onMousedown` and `onMousedownCapture` or are there already other plans?

**Comment by michalochman** [comment]
Thanks for the heads up @zOadT. I was not reviewing future changes to PixiJS yet. This is bad news, because these events properties were trivial to implement and were a perfect way to use events without fear of memory leaks. I guess we’ll just need to convert all event properties to event listeners instead internally in the reconciler and make sure there are no leaks. This should be invisible to users of this library and will not be a breaking change.

**Reply by zOadT** [reply]
Thank you for answering!
